core/trino-main/src/main/java/io/trino/execution/CommentTask.java

```diff
@@ -125,7 +125,7 @@ private void commentOnView(Comment statement, Session session)
             else if (metadata.getTableHandle(session, viewName).isPresent()) {
                 exceptionMessage += ", but a table with that name exists. Did you mean COMMENT ON TABLE " + viewName + " IS ...?";
             }
-            throw semanticException(TABLE_NOT_FOUND, statement, exceptionMessage);
+            throw semanticException(TABLE_NOT_FOUND, statement, "%s", exceptionMessage);
```

Comment:
Let's inline `exceptionMessage` so then you can do formatting at place

Reply:
Done, but only partially. The additional information could also be a format string, but it would require using different `throw`s in each branch, or some other tricks.